packages/melody-plugin-idom/src/visitors/mount.js

```diff
@@ -120,7 +119,6 @@ export default {
                                           args[args.length - 1].value
                                       }"`
                                     : '',
-                                'webpackPrefetch: true',
```

Comment:
We can probably get rid of the array and the `.filter(Boolean).join(',')` and have only:

```javascript
value: path.get('key').is('StringLiteral')
    ? `webpackChunkName: "${args[args.length - 1].value}"`
    : '',
```

What do you think?